index.html

```diff
@@ -61,6 +61,282 @@ <h1>i'm Adan <br>
             </ul>
 
         </section>
+
+        <section class="works">
+            <div class="grid1">
+                <img src="./images/Snapshoot Portfolio.png" alt="snapshot">
+
+            </div>
+            <div class="details">
+                <h1>Tonic</h1>
+
+                <ul class="specific">
+                    <li class="images">CANOPY</li>
+                    <li id="fade">Back End Dev</li>
+                    <li id="fade">2015</li>
+                </ul>
+
+                <p class="brief">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="languages">
+                    <ul class="html-css">
+                        <li>
+                            <a href=""><img src="./images/html.png" class="img2" alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag.png" class="img2" alt=""></a>
+
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag (1).png" class="img2" alt=""></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="see-projects">
+                    <button class="see">See Projects</button>
+                </div>
+
+            </div>
+
+
+
+            <div class="grid3">
+                <img src="./images/Snapshoot Portfolio (1).png" alt="snapshot">
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove all the empty code lines to make in your codebase to make it cleaner and more professional.